src/App.jsx

```diff
@@ -1,15 +1,16 @@
 import React from 'react';
 import './App.css';
+import { withRouter } from 'react-router-dom';
 import ProductOverview from './Components/ProductOverview/ProductOverview';
 import QuestionsAndAnswers from './Components/QuestionsAndAnswers/QuestionsAndAnswers';
 import RelatedProducts from './Components/RelatedProducts/RelatedProducts';
 import Reviews from './Components/Reviews/Reviews';
 
-function App() {
+function App(props) {
+  // console.log('props: ', props.location.pathname);

```

Comment:
@thomHayner commented out this line, but might be using prop.location.pathname to get a product id from the url in the future. keyword: MIGHT.